client/src/Refactor/UploadForm.jsx

```diff
@@ -31,6 +36,16 @@ class UploadScreen extends Component{
     }
   }
 
+  progress(){
+    var xhr = new XMLHttpRequest();
+    xhr.upload.addEventListener("progress", function(evt){
+      if (evt.lengthComputable) {
+        var percentComplete = evt.loaded / evt.total;
+        console.log(percentComplete);
+      }
+    }, false);
+  }
+
   handleClick(event){
     // console.log("handleClick", event);
 
```

Comment:
it's a bit unclear what is going on here, perhaps add some comments for clarification. Not sure what `var self = this;`  on line 52 is doing? If the `console.log(filesArray)` on line 55 is not used for error reporting perhaps remove it? Watch out a linter may complain about not using semicolons on lines 57-60.